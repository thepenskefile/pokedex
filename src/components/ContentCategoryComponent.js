import React from 'react';
// import { Link } from '@reach/router';
import { CONTENT_CATEGORY_TYPES } from '../_types/content_category_types';
import { Tag, Set, Container } from 'fannypack';

const ContentCategoryContainer = () => (
  <Container textAlign="center">
    <Set spacing="major-1">
      {Object.keys(CONTENT_CATEGORY_TYPES).map(category => (
        <Tag key={category}>{CONTENT_CATEGORY_TYPES[category]}</Tag>
      ))}
    </Set>
  </Container>
);

export default ContentCategoryContainer;
