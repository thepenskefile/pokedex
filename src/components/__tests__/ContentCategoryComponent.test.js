import React from 'react';
import ContentCategoryComponent from '../ContentCategoryComponent';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const component = renderer.create(<ContentCategoryComponent />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
