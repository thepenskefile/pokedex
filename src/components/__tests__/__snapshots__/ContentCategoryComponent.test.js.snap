// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="styled-sc-144wh3b-0 iUlbjp Box-sc-6sssmg-0 inpIDs"
  style={
    Object {
      "textAlign": "center",
    }
  }
>
  <div
    className="Flex-sp5dv1-0 InlineFlex-sc-129poqw-0 styled-sc-18btutf-0 gCQOzL Box-sc-6sssmg-0 inpIDs"
    spacing="major-1"
  >
    <div
      className="Flex-sp5dv1-0 InlineFlex-sc-129poqw-0 styled__Tag-sc-1g01kou-2 gysPkN Box-sc-6sssmg-0 inpIDs"
    >
      <div
        className="styled__TagContent-sc-1g01kou-1 iuaGjz Box-sc-6sssmg-0 inpIDs"
      >
        pokemon
      </div>
    </div>
    <div
      className="Flex-sp5dv1-0 InlineFlex-sc-129poqw-0 styled__Tag-sc-1g01kou-2 gysPkN Box-sc-6sssmg-0 inpIDs"
    >
      <div
        className="styled__TagContent-sc-1g01kou-1 iuaGjz Box-sc-6sssmg-0 inpIDs"
      >
        berries
      </div>
    </div>
    <div
      className="Flex-sp5dv1-0 InlineFlex-sc-129poqw-0 styled__Tag-sc-1g01kou-2 gysPkN Box-sc-6sssmg-0 inpIDs"
    >
      <div
        className="styled__TagContent-sc-1g01kou-1 iuaGjz Box-sc-6sssmg-0 inpIDs"
      >
        items
      </div>
    </div>
    <div
      className="Flex-sp5dv1-0 InlineFlex-sc-129poqw-0 styled__Tag-sc-1g01kou-2 gysPkN Box-sc-6sssmg-0 inpIDs"
    >
      <div
        className="styled__TagContent-sc-1g01kou-1 iuaGjz Box-sc-6sssmg-0 inpIDs"
      >
        locations
      </div>
    </div>
    <div
      className="Flex-sp5dv1-0 InlineFlex-sc-129poqw-0 styled__Tag-sc-1g01kou-2 gysPkN Box-sc-6sssmg-0 inpIDs"
    >
      <div
        className="styled__TagContent-sc-1g01kou-1 iuaGjz Box-sc-6sssmg-0 inpIDs"
      >
        machines
      </div>
    </div>
    <div
      className="Flex-sp5dv1-0 InlineFlex-sc-129poqw-0 styled__Tag-sc-1g01kou-2 gysPkN Box-sc-6sssmg-0 inpIDs"
    >
      <div
        className="styled__TagContent-sc-1g01kou-1 iuaGjz Box-sc-6sssmg-0 inpIDs"
      >
        moves
      </div>
    </div>
  </div>
</div>
`;
